src/dns_conf.c

```diff
@@ -573,11 +587,35 @@ static int _conf_domain_rule_ipset(char *domain, const char *ipsetname)
 {
 	struct dns_ipset_rule *ipset_rule = NULL;
 	const char *ipset = NULL;
+	char *copy_name = NULL;
 
 	/* Process domain option */
-	if (strncmp(ipsetname, "-", sizeof("-")) != 0) {
+	if (strncmp(ipsetname, "-", sizeof("-")) == 0) {
+		if (_config_domain_rule_flag_set(domain, DOMAIN_FLAG_IPSET_IGNORE) !=
```

Comment:
ignore should support both ipv4 and ipv6

Reply:
Support rules like `ipset /google.com/#4:-,#6:dns6`?

Reply:
yes